extensions/wikia/PortableInfobox/controllers/PortableInfoboxParserTagController.class.php

```diff
@@ -43,6 +43,31 @@ public static function replaceInfoboxMarkers( &$parser, &$text ) {
 	}
 
 	/**
+	 * @param $markup
+	 * @param Parser $parser
+	 * @param PPFrame $frame
+	 * @return array|string
+	 * @throws UnimplementedNodeException when node used in markup does not exists
+	 * @throws XmlMarkupParseErrorException xml not well formatted
+	 */
+	public function render( $markup, Parser $parser, PPFrame $frame, $params = null ) {
+		$infoboxParser = new Wikia\PortableInfobox\Parser\XmlParser(
+			array_merge( $frame->getNamedArguments(), $frame->getArguments() ) );
```

Comment:
Well - I like how some extra clean up came out of this (separating out method "render")